### PR TITLE
Fixing locator for login page footer

### PR DESCRIPTION
**Pull request by adarshdubey-star**
Locator for login page footer is updated which is causing test failures around components.

### CODE REVIEW

File: airgun/views/login.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 class LoginView(View, ClickableMixin):
     username = TextInput(id="login_login")
     password = TextInput(id="login_password")
-    login_text = Text('//*[@id="login-footer-text"]')
+    login_text = Text(".//footer[contains(@class,'login-pf-page-footer')]")
     logo = Text('//img[@alt="logo"]')
     submit = Text('//button[@type="submit"]')
     version = Text('//*[@id="version"]')
```
Comment (peterdragun) [inline airgun/views/login.py:13]: Any reason not to delete `version` if you stopped using it and there isn't any element with this id?

Comment (shweta83) [reply airgun/views/login.py:13]: @peterdragun Version is removed from UI and it is moved as a part of footer so we are good to remove this @adarshdubey-star

**Review by shweta83 — APPROVED** [review]
Ack

File: airgun/views/login.py (hunk 1 above)
Comment (adarshdubey-star) [reply airgun/views/login.py:13]: Yes, I was going to remove it just wanted to have an ack to remove it. @peterdragun

**Review by peterdragun — APPROVED** [review]

**Review by omkarkhatavkar — APPROVED** [review]

**Pull request merged**